### ISSUE TITLE
Software not hosted under ORNL GH organization are not listed any more

### ISSUE THREAD

**Issue by dalg24** [ORNL/software-catalog#6]
ADIOS, QMCPACK, Cabana, ArborX, Kokkos, etc. are not listed any more in the catalog.

**Comment by aprokop** [comment]
I see ArborX, QMCPACK and Kokkos listed, but not ADIOS and Cabana. I see it under https://ornl.github.io/software-catalog/catalog/. Is that the right site for it?

**Comment by jarrah42** [comment]
Yes that is the correct site. Are ADIOS and Cabana under the ORNL GitHub organization? If not, you will need to let me know where they are and I can add them explicitly.

**Comment by dalg24** [comment]
They are not under ORNL.  They were listed at some point in the past.  You will find multiple hit if you search for them in this repo.  The repo are "ornladios/ADIOS2" and "ECP-copa/Cabana".

"arborx/Arborx" and "kokkos/kokkos" have the same issue (were added in the past, used to be listed but are not showing anymore)

**Comment by aprokop** [comment]
@jarrah42 May also want to add adamantine-sim/adamantine

> "arborx/Arborx" and "kokkos/kokkos" have the same issue (were added in the past, used to be listed but are not showing anymore)

ArborX and Kokkos are showing up.

**Comment by dalg24** [comment]
You're right, I just checked and only Cabana is not showing from the one I had initially reported.

**Comment by aprokop** [comment]
ADIOS2 shows up, but not ADIOS (ornladios/ADIOS). Dunno if both should be there.

**Comment by dalg24** [comment]
ADIOS2 is the right thing.

**Comment by jarrah42** [comment]
So is it only Cabana that is missing?

**Comment by aprokop** [comment]
> So is it only Cabana that is missing?

And adamantine (missing/new) and celeritas (celeritas-project/celeritas)

**Comment by jarrah42** [comment]
I believe this is resolved now. Sorry it took so long!
